pyttsx3/drivers/espeak.py

```diff
@@ -68,7 +74,14 @@ def getProperty(name: str):
         if name == "voices":
             voices = []
             for v in _espeak.ListVoices(None):
-                kwargs = {"id": v.name.decode("utf-8"), "name": v.name.decode("utf-8")}
+                # Use identifier as the unique ID
+                voice_id = v.identifier.decode(
+                    "utf-8"
+                ).lower()  # Identifier corresponds to the "File" in espeak --voices
```

Comment:
Readability: Wrapping lines merely to accommodate a comment makes code harder to read.

Instead, put the comment on the line before.
```suggestion
                # Identifier corresponds to the "File" in espeak --voices
                voice_id = v.identifier.decode("utf-8").lower()
```

Reply:
whoops. I missed this and its a fair point. Ill try and add this in when I merge with the next big ESpeak PR